cmd/atlas/internal/cmdext/cmdext.go

```diff
@@ -391,7 +392,8 @@ func memdir(client *cloudapi.Client, dirName string) (string, error) {
 	if err != nil {
 		return "", err
 	}
-	md := migrate.OpenMemDir(dirName)
+	mdn := "mem://" + dirName
+	md := migrate.OpenMemDir(mdn)
```

Comment:
You should open it without the prefix (mem://).

Reply:
See: https://github.com/ariga/atlas/blob/master/cmd/atlas/internal/cmdapi/migrate.go#L1347

Reply:
done